### PR TITLE
feat: environments throw errors at once

### PR DESCRIPTION
**Pull request by kuyh**
http://git.sphd.io/support/stt-issues/issues/2450

### CODE REVIEW

File: src/utils/env-loader.ts [hunk 1]
```diff
@@ -92,42 +92,52 @@ function loadValueFromEnv(schema: any, object: any, itemKey: string): void {
...
 
-  if (isInvalidEnvValue(defaultValue)) {
-    defaultValue = object[itemKey];
+  // schema-inspector will handle all invalid cases.
+  if (isInvalidEnvValue(loadedValue)) {
+    return;
   }
 
-  if (isInvalidEnvValue(defaultValue) && schema.optional === false) {
-    throw new Error(`Environment "${itemKey}": not optional and has no data`);
```
Comment (sshtel) [inline src/utils/env-loader.ts]: No more error throw in this function now.. is it ok?

Comment (kuyh) [reply src/utils/env-loader.ts]: after this function,
inspector.validate will catch all errors and throw at once

File: src/utils/env-loader.ts [hunk 2]
```diff
@@ -141,12 +151,17 @@ function loadAndSanitize(object: any): void {
...
   const errors = _.filter(result.error || [], v => v.reason !== 'type' && v.property !== '@');
   if (errors.length) {
-    throw new Error(JSON.stringify(errors));
+    let message = '[Environments] Initalization Error!!';
+    _.forEach(errors, err => {
+      message += `\nprocess.env ${err.property} ${err.message}`;
+    });
+    throw new Error(message);
   }
+  return;
```
Comment (Ohen) [inline src/utils/env-loader.ts]: this function was declared `void`. so not necessary `return`

File: src/utils/env-loader.ts [hunk 3]
```diff
@@ -141,12 +151,17 @@ function loadAndSanitize(object: any): void {
...
 }
 
-// Do not execute before sanitize
-function setReadonly(object: any): void {
+// Do not execute before LoadEnv!!!
+export function SetReadonly(object: any): void {
```
Comment (Ohen) [inline src/utils/env-loader.ts]: why changed this function name from `camel case` to `pascal case`?

Comment (kuyh) [reply src/utils/env-loader.ts]: !!! just mistake

**Pull request merged**